app/static/js/event_list.js

```diff
@@ -5,6 +5,34 @@ $(document).ready(function(){
     $("#rsvpBtn").click(function(){
         rsvpForEvent($("#rsvpBtn").val())
     })
+    var viewPastEventsToggle = $("#viewPastEventsToggle");
+  
+
+    viewPastEventsToggle.prop("checked", false);
+    toggleRows(false);  
+
+    viewPastEventsToggle.prop("checked", g_isPastTerm);
+    if (g_isPastTerm){
+      toggleRows(true)
+      viewPastEventsToggle.prop("disabled", true);
+    }
```

Comment:
Lines 11-18 can be replaced with just the prop() lines and one toggleRows call all using `g_isPastTerm`, because you want to pass or set true for all of them when `g_isPastTerm` is true, and false when it is false.